src/classes/STG_InstallScript.cls

```diff
@@ -222,6 +222,22 @@ global without sharing class STG_InstallScript implements InstallHandler {
         }
     }
 
+     /*******************************************************************************************************
+    * @description Disables Preferred Phone handling when upgrading to 1.79
+    * @return void
+    */
+    
+    global static void disablePreferredPhoneEnforcement(Version previousVersion) {
+        if(previousVersion != null && previousVersion.compareTo(new Version(1,79)) < 0) {
+            UTIL_Debug.debug('***Upgrading to version 1.79');
+            Hierarchy_Settings__c orgSettings = Hierarchy_Settings__c.getOrgDefaults();
+            if(orgSettings.Disable_Preferred_Phone_Enforcement__c == null || orgSettings.Disable_Preferred_Phone_Enforcement__c != true) {
+                orgSettings.Disable_Preferred_Phone_Enforcement__c = true;
```

Comment:
This is great, but I think we also want to make sure that the TDTM class is set to Inactive for existing customers as well. After line 80 of this class, we may need to query for the Trigger Handlers and explicitly set the TDTM to false. Might want to confirm with @lifewithryan and others that this is what we should do, but I would expect that the TDTM *must* be set to inactive or we're going to have a lot of unhappy people.

Reply:
@skosgi1 @robertbwatson Yea I think we want to set this to inactive.